src/controllers/room.controller.js

```diff
@@ -0,0 +1,81 @@
+import { Room } from '../models/Room.models.js';
+
+export const getRooms = async (req, res) => {
+  res.send(await Room.findAll());
+};
+
+export const getOneRoom = async (req, res) => {
+  const { id } = req.params;
+  const room = await Room.findByPk(id);
+
+  if (!room) {
+    res.status(404);
+
+    return;
+  }
+  res.status(200);
+  res.send(room);
+};
+
+export const createRoom = async (req, res) => {
+  const { roomName } = req.body;
+
+  if (!roomName) {
+    res.status(400);
+  }
+
+  const room = {
+    roomName,
+  };
+
+  await Room.create(room);
+  res.status(201);
+  res.send(room);
```

Comment:
You are sending the room object that was created locally but not the one that was created in the database. The database might add additional properties to the room object (like an id, createdAt, updatedAt), so it is recommended to send the object returned by the Room.create() method.